backend/server/multi_agent_runner.py

```
@@ -0,0 +1,33 @@
+    """Ensure top-level repo root is importable for multi-agent modules."""
+    repo_root = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", ".."))
+    if repo_root not in sys.path:
+        sys.path.insert(0, repo_root)
+
+
+def _resolve_run_research_task() -> RunResearchTask:
+    _ensure_repo_root_on_path()
+
+    try:
+        from multi_agents.main import run_research_task
+        return run_research_task
```

Ruff (0.15.2)

[warning] Consider moving this statement to an `else` block

(TRY300)